.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
```

Comment:
this kind of feels like a muddled test, since the use strict cases are constantly turning on strict in a non-strict run. Toggling strict is a perf hit, after that it should be faster, but the original point was that if we force strict, we don't need this rule. If we don't, we do need this rule, so in it stays?